src/logic/graphMapping.ts

```diff
@@ -1,36 +1,121 @@
 import {
-  MappedEdge, 
+  MappedEdge,
   MappedNode,
   Edge,
   MappedGraph,
   GraphIndex,
   DisassembledGraph,
+  Position,
 } from './commonInterfaces';
+const rand = require('random-seed');
+
+const SWITCH_ALGORITHM_ABOVE_X_NODES = 10;
+const CIRCLE_RADIUS = 40;
```

Comment:
CIRCLE_RADIUS_PX

Reply:
Its percentage

Reply:
then CIRCLE_RADIUS_PERCENT

Reply:
its the same unit everything is measured in.

Reply:
If I came back to this repo after a long piece of time I would scratch my head add need to look through the repo which unit do we use, either add a comment or  add a `PERCENT`